src/api/Annotations.js

```diff
@@ -118,7 +119,7 @@ export default class Annotations extends MarkerBasedApi {
             },
             file_version: {
                 id: fileVersionId,
-                type: 'file_version',
+                type: FEED_ITEM_TYPE_VERSION,
```

Comment:
Can we revert this change or use a non-feed related constant? The Annotations API module is used outside of the context of Feed.